pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.black]
+line-length = 100
+target-version = ['py310']
+exclude = "(migrations)"
+
+[tool.isort]
+py_version = 310
+profile = "black"
+skip = "migrations"
+
+[tool.pydocstyle]
+convention = "numpy"
+
+[tool.mypy]
+python_version = "3.10"
+allow_untyped_globals = true
+plugins = "mypy_django_plugin.main"
+exclude = "(migrations)"
+follow_imports = "silent"
+
+
+[tool.django-stubs]
+django_settings_module = "meetupselector.settings.dev"
+
+[[tool.mypy.overrides]]
+module = [
+	'celery.*',
+	'debug_toolbar.*',
+	'setuptools.*',
+	'decouple.*',
+]
+ignore_missing_imports = true
```

Comment:
False

Reply:
Why? These libraries don't have stubs, so we have to ignore it (https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker)